.chloggen/fix-ta-security-context.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
let's call this out as a breaking change and note in the subtext why this is also a bug fix.